modules/machineset-yaml-baremetal.adoc

```diff
@@ -0,0 +1,108 @@
+// Module included in the following assemblies:
+//
+// * machine_management/creating-infrastructure-machinesets.adoc
+// * machine_management/creating_machinesets/creating-machineset-bare-metal.adoc
+
+ifeval::["{context}" == "creating-infrastructure-machinesets"]
+:infra:
+endif::[]
+
+:_content-type: REFERENCE
+[id="machineset-yaml-vsphere_{context}"]
```

Comment:
@rohennes hello, while I was doing some bug triaging in the machine mgmt book, I noticed that this ID is inadvertently reused from the vSphere version of this topic. Can you please change it to something unique?